### DISCUSSION TITLE
Creating "step and repeat" banner with montage

### DISCUSSION THREAD

**Discussion by dormant** [ImageMagick/ImageMagick#5361] [Help] [answered]
Is it possible to use the _montage_ command to create a tiled image from two, three or four different images?

I think it's called a "step and repeat" banner. It looks like this:
![stepandrepeat](https://user-images.githubusercontent.com/156362/180614911-19dc9a96-0815-46d3-abaf-094bfd52424a.png)

**Comment by snibgo** [comment, marked as answer]
For example:
```
magick rose: wizard: toes.png -resize 100x100 -duplicate 4,0-1 miff:- | magick montage - -background #579 -gravity North -geometry +75+100 x.png
```
![x](https://user-images.githubusercontent.com/33812368/180616720-99355bfc-f723-4bf3-995e-d896ce54a710.png)

**Comment by dormant** [comment]
That's brilliant. Thanks.

I wrote a small bash script, but this is much easier for playing around.
